src/components/CoinOverview/CoinOverview.js

```diff
@@ -77,9 +81,77 @@ class CoinOverview extends React.Component {
                       obj.price_change_percentage_7d_in_currency
                     ).toFixed(2)}
                   </td>
-                  <td>Test 1</td>
-                  <td>Test 2</td>
-                  <td>Test 3</td>
+                  <td>
+                    <DoubleSpan height="8px" width="100%" background="#2172e5">
+                      <Styled24VMarketCapTextWrapper
+                        width={(obj.total_volume / obj.market_cap) * 100}
+                        background={"white"}
+                      >
+                        <p>
+                          $
+                          {Intl.NumberFormat("en-US", {
+                            notation: "compact",
+                            maximumFractionDigits: 1,
+                          }).format(obj.total_volume)}
+                        </p>
+
+                        <p>
+                          $
+                          {Intl.NumberFormat("en-US", {
+                            notation: "compact",
+                            maximumFractionDigits: 1,
+                          }).format(obj.market_cap)}
+                        </p>
+                      </Styled24VMarketCapTextWrapper>
+                      <SliderWrapper
+                        height="8px"
+                        width="100%"
+                        background="#2172e5"
+                      >
+                        <Slider
+                          width={(obj.total_volume / obj.market_cap) * 100}
+                          background="white"
+                        ></Slider>
+                      </SliderWrapper>
+                    </DoubleSpan>
+                  </td>
+                  <td>
+                    <DoubleSpan height="8px" width="100%" background="#2172e5">
+                      <Styled24VMarketCapTextWrapper
+                        width={(obj.total_volume / obj.market_cap) * 100}
+                        background={"white"}
+                      >
+                        <p>
+                          $
+                          {Intl.NumberFormat("en-US", {
+                            notation: "compact",
+                            maximumFractionDigits: 1,
+                          }).format(obj.circulating_supply)}
+                        </p>
+
+                        <p>
+                          $
+                          {Intl.NumberFormat("en-US", {
+                            notation: "compact",
+                            maximumFractionDigits: 1,
+                          }).format(obj.total_supply)}
+                        </p>
+                      </Styled24VMarketCapTextWrapper>
+                      <SliderWrapper
+                        height="8px"
+                        width="100%"
```

Comment:
is this necessary? do you use this component in another place where width and height is changed?

Reply:
changed. thank you!!